.circleci/initialize-manager.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+from fabric.api import execute
+
+from common import *
+# This is expected to be launch from the ci container
+from ci_variables import *
+
+def initialize_manager(max_runtime, filesystem_timeout):
+    """ Preforms the prerequisite tasks for all CI jobs that will run on the manager instance
+
+    max_runtime (seconds): The maximum uptime this manager should have be for it is terminated
+        This covers potential livelock scenario in one of the jobs (that produces fs activity)
+
+    filesystem_timeout (seconds):  If no fs activity is detected in ~/firesim after 2x this
+        interval, the manager is terminated.  CircleCI provides no mechanism for
+        doing cleanup after all jobs are done (even if some of those jobs fail).
+        So here we use FS activity as a proxy for doneness
+    """
+
+    # Catch any exception that occurs so that we can gracefully teardown the local jjjjjjjjjjjjjjjjjjjjjjj
+    try:
+        put(ci_workdir + "/scripts/machine-launch-script.sh", manager_home_dir)
+        with cd(manager_home_dir):
+            put(ci_workdir + "/scripts/machine-launch-script.sh", ".")
```

Comment:
Why are you re-copying the `machine-launch-script`?

Reply:
Side note: Maybe this is better named `machine-launch-init-script`?

Reply:
Oops, i'll remove that.